[2024-02-22T07:45:58.661+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: count_words_pipeline.push_data_to_kafka_topic manual__2024-02-22T07:45:57.387015+00:00 [queued]>
[2024-02-22T07:45:58.669+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: count_words_pipeline.push_data_to_kafka_topic manual__2024-02-22T07:45:57.387015+00:00 [queued]>
[2024-02-22T07:45:58.670+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-02-22T07:45:58.680+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): push_data_to_kafka_topic> on 2024-02-22 07:45:57.387015+00:00
[2024-02-22T07:45:58.683+0000] {standard_task_runner.py:57} INFO - Started process 281 to run task
[2024-02-22T07:45:58.685+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'count_words_pipeline', 'push_data_to_kafka_topic', 'manual__2024-02-22T07:45:57.387015+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/simple_pipeline.py', '--cfg-path', '/tmp/tmpcbwzr625']
[2024-02-22T07:45:58.687+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask push_data_to_kafka_topic
[2024-02-22T07:45:58.719+0000] {task_command.py:416} INFO - Running <TaskInstance: count_words_pipeline.push_data_to_kafka_topic manual__2024-02-22T07:45:57.387015+00:00 [running]> on host 817e3b351e7c
[2024-02-22T07:45:58.770+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='count_words_pipeline' AIRFLOW_CTX_TASK_ID='push_data_to_kafka_topic' AIRFLOW_CTX_EXECUTION_DATE='2024-02-22T07:45:57.387015+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-22T07:45:57.387015+00:00'
[2024-02-22T07:49:06.401+0000] {local_task_job_runner.py:115} ERROR - Received SIGTERM. Terminating subprocesses
[2024-02-22T07:49:06.431+0000] {process_utils.py:131} INFO - Sending 15 to group 281. PIDs of all processes in the group: [281]
[2024-02-22T07:49:06.439+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 281
